addons/udes_stock/views/product_template.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<odoo>
+    <record id="view_product_template_form_view_udes" model="ir.ui.view">
+        <field name="name">Changes to product form</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view"/>
+        <field name="arch" type="xml">
+            <xpath expr="//page[@name='inventory']" position="inside">
+                <group string="UDES" name="inventory_udes">
```

Comment:
We need to move away from putting `UDES` in the group strings. The system the end user is using is UDES, so they get UDES funtionality. :)

Maybe just `Dimensions`?